azure/appservice-windows/local.tf

```diff
@@ -0,0 +1,5 @@
+locals {
+  auto_tags = {
+      ManageBy: "Terraform"
```

Comment:
Regarding the tags. I'm ok with any decision, but I observed in the previous `tf-service` implementations that we've been using tags in smallcaps. If it was correct, maybe we should comply with the same rule.

Also, instead of 'ManageBy', could it be  'ManagedBy'? (at the end, it's just a flavor, right? I'm ok with both)

Reply:
The tags are in Nmbrs case-sensitive, even though we started using lowercases only.

Reply:
tag name fixed to `ManagedBy` instead.